cmd/cmd.go

```diff
@@ -294,12 +294,12 @@ func CreateNode(numValidators int, chain genesis.ChainType, workingDir string,
 			return nil, nil, err
 		}
 
-		err = config.SaveTestnetConfig(confPath, numValidators)
+		err = config.SaveTestnetConfig(workingDir, confPath, numValidators)
```

Comment:
There is no need to pass the "workingDir". The current directory is set as "workingDir", so any directory can be specified relative to "workingDir".